crates/biome_service/src/projects.rs

```diff
@@ -204,3 +193,23 @@ impl Projects {
         !is_feature_included
     }
 }
+
+#[derive(Clone, Copy, Debug, Deserialize, Eq, Hash, PartialEq, Serialize)]
```

Comment:
I moved this block because it was annoying that the definition was in the middle between the `Projects` definition and its implementation.